Example/EmbeddedCropViewController.swift

```diff
@@ -40,7 +40,7 @@ class EmbeddedCropViewController: UIViewController {
     }
     
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        guard let cropViewController = segue.destination as? CropViewController, let image else {
+        guard let cropViewController = segue.destination as? CropViewController, let image = sender as? UIImage else {
```

Comment:
Why `image = sender`?

Reply:
It's a compile error otherwise and I thought that's how you wanted it to be.

Reply:
I was using [a new feature of Swift 5.7](https://www.hackingwithswift.com/swift/5.7/if-let-shorthand). It was` let image = image`
Now I realized it is not good for users who are not using Swift 5.7.

Can you help me to change it back to `let image = image`? Thanks!